ci/tasks/istio/deploy-istio.sh

```diff
@@ -19,10 +19,16 @@ function install_istio() {
 
     # Install Istio CRDs
     helm template install/kubernetes/helm/istio-init --name istio-init --namespace istio-system | kubectl apply -f -
-    sleep 5
+
+    # Give the CRDs time to apply
+    sleep 10
 
     # Install Istio
-    helm template install/kubernetes/helm/istio --name istio --namespace istio-system | kubectl apply -f -
+    istio_value_overrides="${workspace}/cf-k8s-networking/ci/istio-config/overrides.yml"
+    helm template install/kubernetes/helm/istio --name istio --namespace istio-system --values ${istio_value_overrides} | kubectl apply -f -
+
+    # Enable Istio Sidecar Injection for app workloads
+    kubectl label namespace cf-workloads istio-injection=enabled --overwrite=true
```

Comment:
Not sure where the best place for this to happen actually is. Ideally this would happen when/shortly after the namespace is created. From what I've found, trying to do it in Helm is not considered a best practice either:
https://github.com/helm/helm/issues/5753#issuecomment-502163585

I'm not sure if the sidecar-injector admission controller is able to modify StatefulSets after they've been created, but I doubt it. Would probably need to restart the apps to get them added to existing workloads.

Reply:
It seems a bit surprising to have it in a script called `deploy-istio.sh`.  maybe this script name can change to `deploy-istio-for-eirini`?  or maybe the labeling happens in a subsequent task?

Reply:
Yeah, maybe it should be a task after `configure-eirini-bosh`:
https://github.com/cloudfoundry/cf-k8s-networking/blob/2123a8a242115ded13db19259829fd7ef94ecb88/ci/cf-k8s-pipeline.yml#L330-L336

That's where a lot of `kubectl` steps specific to Eirini happen I believe.

Reply:
making it a two steps process is a good idea. We are not just installing the components for Istio but enabling features for logging.